client/sites/marketing/connections/service.jsx

```diff
@@ -450,27 +447,9 @@ export class SharingService extends Component {
 			// At this point, if there are no available accounts to
 			// select, we must assume the user closed the popup
 			// before completing the authorization step.
-			this.props.failCreateConnection( {
-				message: this.props.translate(
-					'The %(service)s connection could not be made because no account was selected.',
-					{
-						args: { service: this.props.service.label },
-						context: 'Sharing: Publicize connection confirmation',
-					}
-				),
-			} );
```

Comment:
Have we tested these? Is it possible we're not dispatching notices in these cases? This looks like something that belonged to the notices middleware, but we refactored at some point (probably in PRs like https://github.com/Automattic/wp-calypso/pull/49261 and eventually https://github.com/Automattic/wp-calypso/pull/50035 which removed the entire middleware)

Reply:
> Have we tested these? Is it possible we're not dispatching notices in these cases? This looks like something that belonged to the notices middleware, but we refactored at some point (probably in PRs like #49261 and eventually #50035 which removed the entire middleware)

Thus far I've only made changes based on references in code, but I'll try to run through this flow next week to see how it behaves.

Reply:
Sounds good, if we can confirm that we aren't missing to display some notices to the user, I think this PR should be good to go 👍

Reply:
Thanks for the sleuthing, @jsnajdr. Let's use the opportunity to fix these then, rather than deleting them 😉

Reply:
Great find @jsnajdr . Glad that our cleanup could also highlight a real user bug ❤️ 

I'll create a separate pull request to isolate a fix on the missing error notice.

Reply:
Fix PR: #103360